testsuite/MDAnalysisTests/coordinates/test_gro2.py

```diff
@@ -0,0 +1,24 @@
+from MDAnalysis.coordinates.GRO import GROReader, GROWriter
+from MDAnalysisTests.coordinates.base import BaseReference, BaseReaderTest
+from MDAnalysisTests.datafiles import COORDINATES_GRO
+import numpy as np
+
+
+class GROReference(BaseReference):
+    def __init__(self):
+        super(GROReference, self).__init__()
+        self.trajectory = COORDINATES_GRO
+        # self.topology = COORDINATES_GRO
+        self.reader = GROReader
+        self.writer = GROWriter
+        self.ext = 'gro'
+        # self.volume = 0
+        # self.dimensions = np.zeros(6)
+        # self.container_format = True
```

Comment:
I looked into how tests are skipped and I came across the `skipif` decorator that we import from `numpy`.  But then to check the `ext` attribute I'd have to use `self` which I can't. I mean I can't do something like:

`@dec.skipif(self.ext == 'gro')`

Is there a workaround for this? Or should I fall back to vanilla if/else statements in methods,